### ISSUE TITLE
Identical EARFCN assignment for E-UTRA bands 106 and 107

### ISSUE THREAD

**Issue by kuehnhammer** [5G-MAG/Standards#124]
# Context
3GPP TS 36.101 V18.4.0 (2023-12) defines the EARFCN for band 106 in table 5.7.3-1 as:
<img width="337" alt="image" src="https://github.com/5G-MAG/Standards/assets/1220339/dd9d3897-2627-4f6f-ae31-30bc8aab0e06">

... and for band 107 in table 5.7.3H-1 as:
<img width="330" alt="image" src="https://github.com/5G-MAG/Standards/assets/1220339/76254307-0381-4f9b-aaa8-469d2a23f851">

# Problem description
The EARFCNs are identical. If both bands are defined, the eNB cannot determine the intended band from a provided DL EARFCN.

# Suggested solution
Is this by design? If so, how should an LTE-based 5G-BC capable UE handle the situation?

**Comment by jordijoangimenez** [comment]
Hi @kuehnhammer, I'm not sure if this is a problem as the UE will calculate the exact frequency given FDL_low as FDL = FDL_low + 0.1(NDL – NDL_ini), means 612 up to 652 for Band 107; and 470 up to 698. So is the combination of both FDL_low and NDL which uniquely determine the frequency

@haudiobe. What I would suggest is that we include in ETSI TS 103 720 how the UE derives the exact frequency and what the required signaling is to be provided by the TX.

**Comment by kuehnhammer** [comment]
Agreed on the frequency, but how can an eNB know which band to signal in SIB1 based on the configured EARFCN?

**Comment by jordijoangimenez** [comment]
I see your point. I'm not sure how the EARFCN is delivered to the eNBs. The MME should be responsible of managing the SIBs but the SIBs convey information to the UE, not to configure the eNB. The information to the eNB shoudl be carried over M2/M3?

What I think may be needed is a kind of T2-MI (https://dvb.org/wp-content/uploads/2019/12/a136_dvb-t2-modulator-interface.pdf) interface or at least we could describe in ETSI 103 720 how this is done based on the signaling between MME and eNB or how the control plane info from the BM-SC is used to configure the eNBs participating in an MBSFN Service Area.

**Comment by kuehnhammer** [comment]
Seems to be fixed in 18.8.0:

<img width="510" alt="Image" src="https://github.com/user-attachments/assets/1c2069ac-7ae6-4de7-a0ed-f42b6dd5245d" />

**Comment by jordijoangimenez** [comment]
Thank you, @kuehnhammer. I believe we can close the issue.
@johannmika, this spec is now stable, right? In views of updating ETSI TS 103 720 the latest Rel-18 should be taken.
